app/service/rest.py

```diff
@@ -433,8 +433,12 @@ def get_all_notifications_for_service(service_id):
                 notification.job_id,
                 notification.job_row_number,
             )
+            print(f"RECIPIENTE IN service/rest {recipient}")
```

Comment:
Should this be a logging statement instead, or is it just here for debugging purposes for now to test in staging?  Also, minor spelling catch. :-)

```suggestion
            current_app.logger.info(f"RECIPIENT IN service/rest {recipient}")
```

Reply:
ugh it should be removed